workers/unity/Packages/com.improbable.gdk.mobile/Worker/MobileWorkerConnector.cs

```diff
@@ -19,8 +19,9 @@ protected override ConnectionParameters GetConnectionParameters(string workerTyp
                 WorkerType = workerType,
                 Network =
                 {
-                    ConnectionType = NetworkConnectionType.Tcp,
+                    ConnectionType = NetworkConnectionType.Kcp,
                     UseExternalIp = true,
+                    ConnectionTimeoutMillis = 1000,
```

Comment:
we will need to do some tests on what would be a good timeout, but 1s might be too short. let's set it to 10s for now